python/tests/test_table.py

```diff
@@ -10,6 +10,7 @@
 from legate_dataframe.testing import assert_frame_equal, guess_available_mem
 
 
+@pytest.skip(reason="This causes CI hangs. Investigate rewriting this test.")
```

Comment:
```suggestion
@pytest.mark.skip(reason="This causes CI hangs. Investigate rewriting this test.")
```
(Although there is another one in `test_column.py` which we probably also need.)